doc/api/buffer.markdown

```diff
@@ -1358,7 +1636,7 @@ const store = [];
 socket.on('readable', () => {
   var data = socket.read();
   // allocate for retained data
-  var sb = new SlowBuffer(10);
+  var sb = SlowBuffer(10);
```

Comment:
Yep. Let's make those changes separately.
On Mar 18, 2016 11:42 PM, "Сковорода Никита Андреевич" <
notifications@github.com> wrote:

> In doc/api/buffer.markdown
> <https://github.com/nodejs/node/pull/5763#discussion_r56745213>:
>
> > @@ -1358,7 +1636,7 @@ const store = [];
> >  socket.on('readable', () => {
> >    var data = socket.read();
> >    // allocate for retained data
> > -  var sb = new SlowBuffer(10);
> > +  var sb = SlowBuffer(10);
>
> I know this is just a backport so perhaps it should be fixed elsewhere,
> but the SlowBuffer documentation is a bit strange. The documentation says
> that SlowBuffer is a class (a few lines above) and should be called with
> new (a few lines below), but this change removes the new keyword from the
> example code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/5763/files/6c5bc3483031359127069b97889a019b928d7da0#r56745213>
>